test/integration/api/notifications_test.rb

```diff
@@ -13,12 +13,12 @@ class API::NotificationsTest < ActionDispatch::IntegrationTest
     assert_response :ok
   end
 
-  test 'GET /api/notifications/unread.json' do
-    get api_unread_index_path(format: :json)
+  test 'GET /api/notifications.json?status=unread' do
+    get api_notifications_path(query: 'unread', format: :json)
```

Comment:
`query` -> `status` にしたので `status` っぽそう？

```suggestion
    get api_notifications_path(query: 'unread', format: :json)
```